plugins/modules/route53_crrs.py

```diff
@@ -0,0 +1,504 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.amazon.aws.plugins.module_utils.aws.core import (
+    AnsibleAWSModule,
+)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: route53_crrs
+short_description: create, update or delete route53 resource record sets.
+description: create, update or delete route53 resource record sets.
+requirements: [boto3, botocore, python>=3.5]
+options:
+  action:
+    description: >
+      - The action to perform:
+        CREATE  Creates a resource record set that has the specified values.
+        DELETE  Deletes a existing resource record set.
+        UPSERT  If a resource record set doesn't already exist, Route 53
+                creates it. If a resource record set does exist, Route 53
+                updates it with the values in the request.
+    required: true
+    choices: ['CREATE', 'DELETE', 'UPSERT']
+    type: str
+  name:
+    description: >
+      - The name of the record that you want to create, update, or delete.
+    required: true
+    type: str
+  type:
+    description: >
+      - The DNS record type.
+    required: true
+    choices: ['SOA', 'A', 'TXT', 'NS', 'CNAME', 'MX',
+              'NAPTR', 'PTR', 'SRV', 'SPF', 'AAAA', 'CAA']
+    type: str
+  ttl:
+    description: >
+      - The resource record cache time to live (TTL), in seconds.
+    required: true
+    type: int
+  resource_records:
+    description: >
+      - list of the current or new DNS record value(ip address).
+    required: true
+    type: list
+  hosted_zone_id:
+    description: >
+      - The ID of the hosted zone that contains the resource
+        record sets that you want to change.
+    required: true
+    type: str
+  set_identifier:
+    description: >
+      - A unique string to identify each resource record set.
+        it is required if acting upon any of the 'weight', 'region',
```

Comment:
```suggestion
        it is required if acting upon any of the I(weight), I(region),
```
Use `I(...)` for option names, `C(...)` for option values.